agora/shopper_insights_api/src/video_processor.py

```diff
@@ -26,8 +29,30 @@ def __init__(self, url, index, name, debug=False):
         self.inactivity_threshold = 30  # 30 seconds
```

Comment:
myabe we should add a constructor flag that enables/disables rcording

Reply:
added a flag in the constructor to do this